assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6

```diff
@@ -119,6 +119,13 @@ export default Ember.Controller.extend({
       this.save().then(() => this.send('run'));
     },
 
+    sortByProperty(property) {
+      if (this.sortBy == `${property}:desc`){
```

Comment:
We should use `===` here. Configure [ESLint](https://eslint.org) in your code editor to run on save.

Also for consistent formatting (notice `() {` vs `(){` above), I recommend configuring your code editor to run [Prettier](https://prettier.io/) on save.

Reply:
I tried `===` at first, then I realised that `===` can never return true for arrays created in different locations, even if their contents are the same. In this case, I think we can only use `==`. 
More info here: https://stackoverflow.com/a/13757153

Yes, I've set up Prettier and ESLint now :+1:

Reply:
Why are we comparing an array with a string? This should be:

```
(this.sortBy[0] === `${property}:desc`)
```

Reply:
Wow, my bad, this is much better :disappointed: 
Fixed :+1: